### PR TITLE
improvement: make tika server failures more informative

### LINKED ISSUE #34: Make Tika Server failure messages more informative
Currently, when their is an issue with external Tika Server, the task logs just tell us that metadata extraction is failing because of a server issue, they don't specify any further details about the type of issue, for example, HTTP status code returned.

This makes it difficult to rectify issues with an external Tika Server.

Recommend adding more informative messages to exceptions thrown, based on status codes returned.

### PR DESCRIPTION
**Pull request by tomdickman**
Resolves #34

### CODE REVIEW

**Comment by tomdickman** [comment]
This is blocked by #36 PR as a refactor will be required before merge is possible.

**Review by kristian-94 — APPROVED** [review]

**Comment by tomdickman** [comment]
This has been resolved in PR #36

**Pull request closed without merging**
